### DISCUSSION TITLE
How to make Ruby Solargraph "less strict"

### DISCUSSION THREAD

**Discussion by ianyepan** [emacs-lsp/lsp-mode#3471] [Q&A] [answered]
I've set up ruby-mode to use solargraph and LSP. Currently, lsp-ui is displaying a ton of trivial warnings like "extra blank lines detected", "unnecessary blank lines", "don't use parentheses" etc. 

Flycheck shows that these trivial reminders are of "**info**" level, and I'd like "**warning**" and "**error**" to be shown in lsp-ui.

I note that there is "flycheck-error-list-minimum-level", but it doesn't seem to affect lsp-ui sideline.

# Simple example screenshot

![image](https://user-images.githubusercontent.com/36686900/163691617-1b3f82ce-9a7b-4966-be04-7517fdb67093.png)

**Comment by elken** [comment, marked as answer]
Nothing to do with emacs, customize [rubocop](https://docs.rubocop.org/rubocop/index.html) to your heart's content
